tests/installation/ipxe_install.pm

```diff
@@ -148,8 +158,27 @@ sub enter_o3_ipxe_boot_entry {
 }
 
 sub set_bootscript_cmdline_extra {
-
     my $cmdline_extra = " ";
+    if (is_sle('>=16')) {
+        if (get_var('AGAMA_AUTO')) {
+            my $agama_auto = data_url(get_var('AGAMA_AUTO'));
+            $agama_auto =~ s/^\s+|\s+$//g;
+            $cmdline_extra .= "agama.auto=$agama_auto ";
+        }
+        # Agama Installation repository URL
+        # By default Agama installs the packages from the repositories specified in the product configuration.
+        # From now Agama supports using the agama.install_url boot parameter for overriding the default installation repositories.
+        if (get_var('AGAMA_INSTALL_URL')) {
+            my $agama_install_url = get_var('AGAMA_INSTALL_URL');
+            $agama_install_url =~ s/^\s+|\s+$//g;
+            $cmdline_extra .= "agama.install_url=$agama_install_url ";
+        }
+        if (is_ipmi) {
+            my $sol_console = is_aarch64 ? get_var('SERIALDEV', 'ttyAMA0') : get_var('SERIALDEV', 'ttyS1');
```

Comment:
Console information is should be already part of `IPXE_CONSOLE` and you can extract tty from it.